src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.tsx

```diff
@@ -34,11 +35,16 @@ const AdvancedSearchResultDetailsList: React.FC<IAdvancedSearchResultDetailsList
 }) => {
     const { t } = useTranslation();
     const twinCount = twins.length;
-
     const classNames = getClassNames(styles, {
         theme: useTheme()
     });
-    const columns: IColumn[] = [
+    const sortKey = '$dtId';
```

Comment:
the sort key needs to live in state so that it can be updated and triggers renders when changed.

Reply:
Ignore this ^ @msnyder-msft . I understand now